### ISSUE TITLE
Running Python in shell does not find installed packages

### ISSUE THREAD

**Issue by slhck** [sdispater/poetry#1604]
- [x] I am on the [latest](https://github.com/sdispater/poetry/releases/latest) Poetry version.
- [x] I have searched the [issues](https://github.com/sdispater/poetry/issues) of this repo and believe that this is not a duplicate.
- **OS version and name**: macOS 10.14
- **Poetry version**: 0.12.17
- **Link of a [Gist](https://gist.github.com/) with the contents of your pyproject.toml file**: https://gist.github.com/slhck/5a5dc06d77046a894025d593447bfdc8

## Issue

I want to use `poetry shell` to run `python3` with the packages installed via `poetry`.

When I install the packages and run `python3` from the shell, the packages are not found:

```
➜ poetry init
…
~/Desktop/test via 🐍 3.7.3
➜ poetry add fastapi uvicorn
Creating virtualenv test-py3.7 in /Users/werner/Library/Caches/pypoetry/virtualenvs
Using version ^0.42.0 for fastapi
Using version ^0.10.8 for uvicorn

Updating dependencies
Resolving dependencies... (4.9s)

Writing lock file


Package operations: 9 installs, 0 updates, 0 removals

  - Installing click (7.0)
  - Installing h11 (0.8.1)
  - Installing httptools (0.0.13)
  - Installing pydantic (0.32.2)
  - Installing starlette (0.12.9)
  - Installing uvloop (0.14.0)
  - Installing websockets (8.1)
  - Installing fastapi (0.42.0)
  - Installing uvicorn (0.10.8)

~/Desktop/test via 🐍 3.7.3 took 19s
➜ poetry run python3 -c "import fastapi"
email-validator not installed, email fields will be treated as str.
To install, run: pip install email-validator

~/Desktop/test via 🐍 3.7.3 took 2s
➜ poetry shell
Spawning shell within /Users/werner/Library/Caches/pypoetry/virtualenvs/test-py3.7

~/Desktop/test via test-py3.7 via 🐍 3.7.3
➜ python3 -c "import fastapi"
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'fastapi'
```

The module is not found when inside the shell, but it is when I use `poetry run`.

Python 3 is 3.7.3 installed via `pyenv`:

```
➜ pyenv versions
  system
  3.6.5
* 3.7.3 (set by PYENV_VERSION environment variable)
```

Unless I've missed something obvious, I think this is the way this tool is intended to be used. I also know `pipenv shell` does the same, and from the documentation I don't see any indication that the expected behavior would be different.

**Comment by enkidulan** [comment]
Same here, but on Linux (Fedora 31). Seems like `poetry shell` command does not activate virtual environment for the shell session.

**Comment by enkidulan** [comment]
Also none of the packages' console scritps are avalivable in the spawned session

**Comment by slhck** [comment]
I just tried 1.0.0b5 and it works there.

**Comment by enkidulan** [comment]
same here, fixed in `1.0.0b5`.
@slhck thanks!

**Comment by finswimmer** [comment]
Thanks a lot for coming back and reporting! :+1: 

So I will close this issue.
